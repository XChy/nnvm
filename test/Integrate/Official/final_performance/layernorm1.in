version https://git-lfs.github.com/spec/v1
oid sha256:de7b31cfd4d9cc87582b5064c4321ee2698c341fdc4efe23edad7615d420e3f8
size 5906141
