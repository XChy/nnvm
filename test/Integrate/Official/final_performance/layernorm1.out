-0x1.557dp-8
0